lib/sport_ngin_aws_auditor/convenience_wrappers.rb

```diff
@@ -1,54 +1,85 @@
 require_relative './aws'
+require 'aws-sdk'
 require_relative './google'
 
 module SportNginAwsAuditor
-  attr_accessor :creds
+  attr_accessor :assume_role_creds
 
   module AWSWrapper
-    attr_accessor :aws, :account_id
-
-    def aws(environment, roles)
-      if roles
-        SportNginAwsAuditor::AWSSDK.authenticate_with_roles(environment)
+    def aws(environment, global_options)
+      if global_options[:aws_roles]
+        SportNginAwsAuditor::AWSSDK.update_aws_config({region: 'us-east-1'})
+      elsif global_options[:assume_roles]
+        @assume_role_creds = SportNginAwsAuditor::AWSSDK.authenticate_with_assumed_roles(environment,
+                                                                                         global_options[:arn_id],
+                                                                                         global_options[:role_name],
+                                                                                         get_sts_client(environment))
```

Comment:
@bbergstrom We ran into issues with that when attempting to run different accounts in a row. Apparently the clients are cached with specific regions, so it had trouble going back and forth between regions like that.